concent_api/core/tests/test_payments_bankster.py

```diff
@@ -268,20 +268,22 @@ def test_that_settle_overdue_acceptances_should_return_none_if_subtask_costs_whe
             )
         ]
 
-        with mock.patch(
-            'core.payments.bankster.service.get_list_of_payments',
-            side_effect=[
-                self._get_list_of_batch_transactions(),
-                self._get_list_of_force_transactions(),
-            ]
-        ) as get_list_of_payments_mock:
-            claim_against_requestor = settle_overdue_acceptances(
-                requestor_ethereum_address=task_to_compute.requestor_ethereum_address,
-                provider_ethereum_address=task_to_compute.provider_ethereum_address,
-                acceptances=subtask_results_accepted_list,
-                requestor_public_key=hex_to_bytes_convert(task_to_compute.requestor_public_key),
-            )
+        with mock.patch('core.payments.bankster.validate_list_of_transaction_timestamp', return_value=None) as validate_list_of_transaction_mock:
```

Comment:
Is there any test that tests this newly created function `validate_list_of_transaction_timestamp`?

Reply:
To my surprise there is in `test_integration_force_payment.py`: `test_provider_send_force_payment_beyond_payment_time_concent_should_reject()`

Reply:
There's no corner case being tested where there is `closure_time` > `youngest_payments_ts`, but then it's business as usual :P

Reply:
@dybi Do you think we should have separate unit test for this function? If not, please resolve this conversation.